backend/src/main/scala/bloop/tracing/TraceProperties.scala

```diff
@@ -0,0 +1,38 @@
+package bloop.tracing
+
+import scala.util.Properties
+
+case class TraceProperties(
+    serverUrl: String,
+    // Note: `debug` corresponds to a sampling flag sent to Zipkin server,
+    // while `verbose` determines if we send certain spans marked as verbose spans
+    debug: Boolean,
```

Comment:
Should we call it `enableZipkinDebug` to try to make the distinction clearer for users that don't have access to the source?

Reply:
I like that, let's leave that to a separate PR perhaps?